ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vlan-interfaces.j2

```diff
@@ -76,6 +76,12 @@ interface {{ vlan_interface }}
 {%         if vlan_interfaces[vlan_interface].ospf_area is defined %}
    ip ospf area {{ vlan_interfaces[vlan_interface].ospf_area }}
 {%         endif %}
+{%         if vlan_interfaces[vlan_interface].ospf_message_digest is defined and vlan_interfaces[vlan_interface].ospf_message_digest is not none %}
+   ip ospf authentication message-digest
+{%           for id in vlan_interfaces[vlan_interface].ospf_message_digest %}
```

Comment:
Please sort the id as per below, to match cli output when multiple ids are provided.
{% for id in vlan_interfaces[vlan_interface].ospf_message_digest | sort %}

Reply:
Updates as requested.